curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/inpaint.html

```diff
@@ -0,0 +1,103 @@
+<html>
+<head>
+<meta charset="utf-8">
+<title>CVdrawing</title>
+</head>
+<body>
+<p id="status">OpenCV.js is loading...</p>
+<canvas id="canvas" >
+</canvas>
+<img id="src" src="bird.jpg" style="display:none"/>
+<input type="button" id="myButton" value="Test" disabled=true
+onclick="inpaint()"/>
+
+<script type="text/javascript">
+
+var src;
+var mask;
+var lastX;
+var lastY;
+var x;
+var y;
+
+window.onload = function()
+{
+context = document.getElementById("canvas").getContext("2d");
+img = document.getElementById("src");
+context.canvas.width = img.width;
+context.canvas.height = img.height;
+context.drawImage(img, 0, 0);
+};
+
+document.getElementById("canvas").onmousedown = function(e)
+{
+  x = e.pageX - this.offsetLeft;
+  y = e.pageY - this.offsetTop;
+  if (lastX == undefined)
+  {
+      lastX = x;
+      lastY = y;
+  }
+};
+
+document.getElementById("canvas").onmousemove = function(e)
+{
+  if(lastX != undefined)
+  {
+    cv.line(src, new cv.Point(lastX, lastY), new cv.Point(x, y),
+    new cv.Scalar(255, 255, 255, 255), 2);
+    cv.line(mask, new cv.Point(lastX, lastY), new cv.Point(x, y),
+    new cv.Scalar(255, 255, 255, 255), 2);
+    lastX = x;
+    lastY = y;
+    x = e.pageX - this.offsetLeft;
+    y = e.pageY - this.offsetTop;
+    cv.imshow("canvas", src);
+  }
+};
+
+document.getElementById("canvas").onmouseup =
+document.getElementById("canvas").onmouseleave = function(e)
+{
+    lastX = undefined;
+    lastY = undefined;
+};
+
+function init()
+{
+  src = cv.imread("src");
+  let cols = src.size().width;
+  let rows = src.size().height;
```

Comment:
unused variables?

Reply:
Indeed, we do not using them for mask size.
Variables cols, rows are deleted.